app/controllers/webhooks_controller.rb

```diff
@@ -8,8 +8,13 @@ def receive
     organization = Maestrano::Connector::Rails::Organization.find_by_uid(org_uid)
     if organization
       Rails.logger.debug("WebhooksController.receive with params: #{webhook_params}")
-      entities_hash ={params[:entity].singularize.capitalize => [webhook_params]}
-      Maestrano::Connector::Rails::PushToConnecJob.perform_later organization, entities_hash
+      if(params[:entity] === 'orders')
```

Comment:
@Berardpi can you think of any better way of processing Orders notifications so we push both a SalesOrder and an Invoice to Connec! ?